cmd2/cmd2.py

```diff
@@ -146,16 +134,6 @@ def categorize(func: Union[Callable, Iterable], category: str) -> None:
     else:
         setattr(func, HELP_CATEGORY, category)
 
-
-def _which(editor: str) -> Optional[str]:
```

Comment:
Thanks for moving this to **utils.py**, that is a more appropriate location for it